nixos/modules/virtualisation/containers.nix

```diff
@@ -694,7 +694,7 @@ in
     # container so that container@.target can get the container
     # configuration.
     environment.etc =
-      let mkPortStr = p: p.protocol + ":" + (toString p.hostPort) + ":" + (if p.containerPort == null then toString p.hostPort else toString p.containerPort); 
+      let mkPortStr = p: p.protocol + ":" + (toString p.hostPort) + ":" + (if p.containerPort == null then toString p.hostPort else toString p.containerPort);
```

Comment:
Even if I see that this changes nothing, I think it is better for tractability to leave this line unchanged (at least in the context of postgresql updates) (I guess you have automated white space stripping in your text editor).